layouts/partials/sidebar-tree.html

```diff
@@ -0,0 +1,80 @@
+{{/*
+
+  A modified version of the siderbar-tree.html partial in Docsy to add the ToC.
```

Comment:
The omission of the comment is not deliberate. I tried to document the change with the "to add the ToC" part. I thought it was clear enough since the partial is called `toc.html`.

Reply:
I restored the comment and clarified the modification.